apps/auracast/syscfg.yml

```diff
@@ -59,4 +59,6 @@ syscfg.vals:
     BLE_MAX_BIG: 1
     BLE_MAX_BIS: 2
 
-    BLE_PHY_NRF52_HEADERMASK_WORKAROUND: 1
\ No newline at end of file
+    BLE_PHY_NRF52_HEADERMASK_WORKAROUND: 1
```

Comment:
this is only required on nrf52 if encryption is enabled

Reply:
I'll remove it then